src/hooks/useRemovableFilters.ts

```diff
@@ -1,5 +1,5 @@
 import { FieldValueFilter, useSearchState, useSearchActions, DisplayableFacet, DisplayableFacetOption, SearchActions, Matcher } from '@yext/search-headless-react';
-import { isEqual } from 'lodash';
+import lodash from 'lodash';
```

Comment:
I'd use lodash-es instead.

Reply:
I attempted to use lodash-es, but I was running into some strange errors with jest, however regular lodash worked fine so I stuck with that